lib/PAUSE/package.pm

```diff
@@ -856,17 +729,18 @@ sub checkin {
 
   if ($row) {
       # We know this package from some time ago
-      $self->update_package($row);
+      $self->update_package($ctx, $row);
   } else {
       # we hear for the first time about this package
-      $self->insert_into_package;
+      $self->insert_into_package($ctx);
   }
 
-  my $status = $self->get_index_status_status;
-  if (! $status or $status == PAUSE::mldistwatch::Constants::OK) {
-      $self->checkin_into_primeur; # called in void context!
-  }
+  # my $status = $self->get_index_status_status($ctx);
```

Comment:
Why are tehse conditionals commented out? Because we won't reach this if the contexts abort? Maybe remove them or add a comment?

Reply:
Thanks, I've removed them.